features/comparisons.feature

```diff
@@ -318,6 +341,26 @@ Feature: comparisons
         [ 10                         , '>'  , DecimalType.new(5)         , true  ]  ,
         [ 10                         , '>'  , DecimalType.new(10)        , false ]  ,
 
+        # Groups
+        [ Temperatures               , '<'  , Temperatures.max           , true  ]  ,
```

Comment:
These tests should be bidirectional, right?

Reply:
Yes, most of them at least, do we e.g. support String <=> Quantity type?

Reply:
I will create an issue for this, I would especially expect SwitchItem == OtherSwitchItem to work, and the other ones need to be discussed and tests created. Feels like this is outside the scope of this PR though.